nncf/common/graph/module_attributes.py

```diff
@@ -10,23 +10,70 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
-
+from abc import ABC
+from abc import abstractmethod
+from enum import Enum
+from typing import List
 from typing import Tuple
 
 
-class BaseModuleAttributes:
+class Dtype(Enum):
+    FLOAT = 'float'
+    INTEGER = 'int'
+
+
+class BaseModuleAttributes(ABC):
```

Comment:
Please rename BaseModuleAttributes to BaseLayerAttributes according to the Style Guide.

Reply:
Done